std_msgs/Header header
int8 mode
bool brew
